src/main/java/org/candlepin/resteasy/ResourceLocatorMap.java

```diff
@@ -234,7 +240,8 @@ protected void logLocators() {
                 continue;
             }
 
-            if (!m.isAnnotationPresent(Consumes.class)) {
+            if (!m.isAnnotationPresent(Consumes.class) &&
```

Comment:
Tests are hitting NPE at line 255. I guess we have to seach for the interface annotation as well.